tests/core/basic20/task.yaml

```diff
@@ -1,6 +1,6 @@
 summary: Check basic core20 system functionality
 
-systems: [ubuntu-core-20-*]
+systems: [ubuntu-core-2*]
```

Comment:
thanks for this, we should check for all `ubuntu-core-20` system regexps to be updated to `ubuntu-core-2*` so when it comes time for core22, we will get as much test coverage day 1 as possible